codeclimate-pylint

```diff
@@ -15,6 +15,12 @@ include_paths = config.get('include_paths', ['./'])
 exclude_paths = config.get('exclude_paths', [])
 plugins_config = config.get('plugins', [])
 
+additional_python_path = config.get('PYTHONPATH', [])
+if isinstance(additional_python_path, str):
+    additional_python_path = [additional_python_path]
+sys.path.extend(map(lambda x: os.path.abspath(x), additional_python_path))
+print("Running pylint using PYTHONPATH: " + (":".join(sys.path)), file=sys.stderr)
```

Comment:
You'll have to remove this `print` statement too since CodeClimate expects only a valid JSON with the engine's output in `stdout`

Reply:
Yep. That's why this is printed to stderr, so it does not affect the JSON output, and can be used during trroubleshooting with CODECLIMATE_DEBUG=1

Reply:
Oh indeed! Sorry for that.

Reply:
:) no problem, thanks again!